src/main/resources/OSGI-INF/stores-contrib.xml

```diff
@@ -0,0 +1,7 @@
+<component name="org.nuxeo.wopi.stores">
+
+  <extension target="org.nuxeo.runtime.kv.KeyValueService" point="configuration">
+    <store name="wopi-locks" class="org.nuxeo.runtime.kv.MemKeyValueStore"/>
```

Comment:
This will need to be easily changeable when in cluster. `nuxeo-s3-direct-upload` has the same problem I see... We need a better template/fallback mechanism for the k/v stores...

Reply:
We probably won't need the `KeyValueStore` anyway as we need to handle ourselves the `ttl` to clean up locked documents.
If we stick to the `KeyValueStore`, that contribution will be removed to use the default one, praying that no conflict will happen (it probably won't as we will use the WOPI file ID as key).

Reply:
OK, we have a User Story about locks, we might not be using the KeyValueStore anymore, see https://jira.nuxeo.com/browse/NXP-25382